providers/catalog.go

```diff
@@ -278,11 +279,40 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 		},
 	},
 
+	// Klaviyo configuration
+	Klaviyo: {
+		AuthType: Oauth2,
+		BaseURL:  "https://a.klaviyo.com/api",
```

Comment:
There's some endpoints that require `client` instead of `api`, so can you please update this to be just `https://a.klaviyo.com`?